<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PersonalCardReader 1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Выход</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Start">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Excel_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Сотрудники</string>
    </property>
   </widget>
   <widget class="QLabel" name="PersonalName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Excel_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Мониторинг</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>376</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
